validate.js

```diff
@@ -0,0 +1,14 @@
+const form = document.forms[0];
+const email = document.getElementById('email');
+const error = document.getElementById('error');
+const message = document.querySelector('.error-message');
+form.addEventListener('submit', (event) => {
+  const messageWhileError = [];
+  if (email.value.match(/[A-Z]/)) {
+    messageWhileError.push('Email should be writen in small leter.');
```

Comment:
- [ ] Great job validating your form 💯, but as required in the project, the error message should be close to the submit button. Kindly, relocate it. 🔍 

![image](https://user-images.githubusercontent.com/91301423/203769826-42b899bd-6950-48de-ad9d-009251aa93f6.png)